text/0002-keyboard-accessibility.md

```diff
@@ -0,0 +1,50 @@
+- Start Date: March 16, 2023
+- RFC PR: (leave this empty)
+- Issue: (leave this empty)
+
+# Summary
+
+A quick summary of accessibility principles we use for Webstudio. Full guide is https://www.w3.org/WAI/ARIA/apg/patterns/
+
+# Detailed design
+
+## Which elements should be focusable?
+
+All interactive elements should be focusable by keyboard. Interactive elements are those that do anything user-facing on mouse or keyboard interactions, like click or enter.
+
+Example: A button or a link should always be focusable, whereas a div that has no interactive usage not be focusable, unless it is a list item and has focusable elements inside.
+
+## Vertical lists of focusable items
+
+Long lists with focusable items are hard to exit if we tab through items. This is why once the first list element is focused, we choose to use arrow keys to navigate focus in the list.
```

Comment:
What do we do in the following situation?
- User used arrow navigation in the list, and  focused 5th item
- User clicked tab, so the next element out of the list focused
- User clicked shif-tab, do we restore the focus position on the 5th element in the list or focus first?

Reply:
good question, my first instinct would be to focus the first item just like we entered the first time the list, but focusing last focused element wouldn't be totally wrong either if there is a good way to remember that and make it work consistently